tests/sqlparser_common.rs

```diff
@@ -8202,17 +8202,10 @@ fn parse_position() {
 
 #[test]
 fn parse_position_negative() {
-    let sql = "SELECT POSITION(foo) from bar";
```

Comment:
Should we add this case to valid test as well?

Reply:
It's syntactically valid, but I don't think any DBMS accepts it this so I'm prefer not to add a test for it.